sale_commission/models/settlement.py

```diff
@@ -181,7 +182,7 @@ class SettlementLine(models.Model):
         related='settlement.company_id',
```

Comment:
@pedrobaeza in v11 this has `readonly=True`. Isn't it needed here?

Reply:
No, this is the default in v12, and thus, not needed.